src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/JavaCallableWrapperGenerator.cs

```diff
@@ -574,8 +576,7 @@ public void Generate (TextWriter writer)
 
 				if (children != null) {
 					for (int i = 0; i < children.Count; ++i) {
-						string methods = string.Format ("__md_{0}_methods", i + 1);
-						GenerateRegisterType (writer, children [i], methods);
+						GenerateRegisterType (writer, children [i], $"__md_{i + 1}_methods");
```

Comment:
While I can see multiple `.Write()` calls being faster -- and that makes up the majority of this PR -- are format strings actually faster than `string.Format()`?

For a quick one-off, it looks like the IL for these two statements is identical:

```csharp
int i = 1;
var s1 = string.Format ("__md_{0}_methods", i);
var s2 = $"__md_{i}_methods";
```

Both `s1` and `s2` become `string.Format()` calls.  Thus, this doesn't appear to be a significant change.

Reply:
@jonathanpeppers wrote:

> Were you testing .NET 7?

Oops.  Nope. :-(